tailscale.h

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//
+// Tailscale C library.
+//
+// Use this library to compile Tailscale into your program and get
+// an entirely userspace IP address on a tailnet.
+//
+// From here you can listen for other programs on your tailnet dialing
+// you, or connect directly to other services.
+//
+
+
+#include <stddef.h>
+
+// tailscale is a handle onto a Tailscale server.
+typedef int tailscale;
```

Comment:
If you want stronger type safety on these (no implicit conversions to int), use a pointer to a struct that doesn't have its members declared in the `.h`. The similarity to fds is kind of nice though.